src/pages/clouddocs/annotator_smoking.md

```diff
@@ -5,19 +5,13 @@ categories: Annotators
 slug: annotator_smoking
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023               -->
+<!--                                                                    -->
+<!-- SPDX-License-Identifier: Apache-2.0                                -->
+<!--                                                                    -->
 
-copyright:
-  years: 2011, 2021
-lastupdated: "2019-09-21"
-
-keywords: annotator clinical data, clinical data, annotation
-
-subcollection: wh-acd
-
----
-
-# Smoking (Deprecated) -->
+<!-- # Smoking -->
```

Comment:
Is this one still deprecated or not? Doesn't really matter in the comment, but helpful to keep consistent with what is presented, and that shows deprecated today.